### DISCUSSION TITLE
Orderbook histoy

### DISCUSSION THREAD

**Discussion by isslerman** [ocignis/tradezap#1] [Ideas]
First thank you for the script.
 It's working and is a nice help to download some Binance data. 

Do you know if it is possible to download orderbook history from somewhere? kdepth. 
Binance only allow to download history from VIP users. 

Tks again! 
Marcos Issler

**Comment by mkosir** [comment]
Hi @isslerman, glad you find this CLI package useful!
As you mentioned, also didn't find any way to integrate it into package, but if you find some nice approach don't hesitate to include it in this package!
